Cdnfsd_TransparentCDN_Popup_View_Success.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace W3TC;
+
+if ( !defined( 'W3TC' ) )
+	die();
+?>
+<form class="w3tc_cdn_transparentcdn_fsd_form">
+    <div class="metabox-holder">
+        <?php Util_Ui::postbox_header(
+	__( 'Succeeded', 'w3-total-cache' ) ); ?>
+
+        <div style="text-align: center">
+            <?php __('Site was successfully configured.', 'w3-total-cache') ?> <br />
```

Comment:
Using `__()` without `echo` will not print the string.  You can use `_e()` or `echo __()` to print the string.  Please see https://developer.wordpress.org/reference/functions/__/#user-contributed-notes